src/Instance/ItemsList/ItemsList.js

```diff
@@ -107,6 +107,35 @@ const visibleColumns = [
 const dragVisibleColumns = ['dnd', 'select', ...visibleColumns];
 const rowMetadata = ['id', 'holdingsRecordId'];
 
+const DropZone = ({
+  isItemsDropable,
+  children,
+  droppableId,
+  isDropDisabled,
+}) => {
+  return isItemsDropable ? (
+    <Droppable
+      droppableId={droppableId}
+      isDropDisabled={isDropDisabled}
```

Comment:
just `isDropDisabled` ?

Reply:
We don't need Drappable at all if items are not droppable

Reply:
ah, I missed it's another value